share/default

```diff
@@ -105,8 +105,8 @@ CharView.Menu.Point.HVCurve:Ctrl+5
 +CharView.Menu.Point.G2 Curve:Ctrl+1
 +CharView.Menu.Point.G4 Curve:Ctrl+2
 +CharView.Menu.Point.Corner:Ctrl+3
-+CharView.Menu.Point.Prev Constraint:Ctrl+4
-+CharView.Menu.Point.Next Constraint:Ctrl+5
```

Comment:
It is not the bare numbers @skef&hellip;

Reply:
Those are different controls directly on the Point menu. They convert selected points between those types. The tools submenu switches between palette tools (a more common operation, so a sensibly simpler key combo).